app/services/ai/clients/index.js

```diff
@@ -10,7 +10,7 @@ const clients = {
 }
 
 const getClient = (modelId) => {
-  const [vendor, service, model] = modelId.split(':')
+  const [vendor, service, model] = modelId.split('|')
 
   const client = clients[`${vendor}:${service}`]
```

Comment:
This needs updating to `|`

Reply:
I've changed the : to | in client/clients